src/core/cff_parser.js

```diff
@@ -108,38 +108,38 @@ const CFFStandardStrings = [
 const NUM_STANDARD_CFF_STRINGS = 391;
 
 const CharstringValidationData = [
-  null,
-  { id: "hstem", min: 2, stackClearing: true, stem: true },
-  null,
-  { id: "vstem", min: 2, stackClearing: true, stem: true },
-  { id: "vmoveto", min: 1, stackClearing: true },
-  { id: "rlineto", min: 2, resetStack: true },
-  { id: "hlineto", min: 1, resetStack: true },
-  { id: "vlineto", min: 1, resetStack: true },
-  { id: "rrcurveto", min: 6, resetStack: true },
-  null,
-  { id: "callsubr", min: 1, undefStack: true },
-  { id: "return", min: 0, undefStack: true },
-  null, // 12
-  null,
-  { id: "endchar", min: 0, stackClearing: true },
-  null,
-  null,
-  null,
-  { id: "hstemhm", min: 2, stackClearing: true, stem: true },
-  { id: "hintmask", min: 0, stackClearing: true },
-  { id: "cntrmask", min: 0, stackClearing: true },
-  { id: "rmoveto", min: 2, stackClearing: true },
-  { id: "hmoveto", min: 1, stackClearing: true },
-  { id: "vstemhm", min: 2, stackClearing: true, stem: true },
-  { id: "rcurveline", min: 8, resetStack: true },
-  { id: "rlinecurve", min: 8, resetStack: true },
-  { id: "vvcurveto", min: 4, resetStack: true },
-  { id: "hhcurveto", min: 4, resetStack: true },
-  null, // shortint
-  { id: "callgsubr", min: 1, undefStack: true },
-  { id: "vhcurveto", min: 4, resetStack: true },
-  { id: "hvcurveto", min: 4, resetStack: true },
+  /* 0 */ null,
+  /* 1 */ { id: "hstem", min: 2, stackClearing: true, stem: true },
+  /* 2 */ null,
+  /* 3 */ { id: "vstem", min: 2, stackClearing: true, stem: true },
+  /* 4 */ { id: "vmoveto", min: 1, stackClearing: true },
+  /* 5 */ { id: "rlineto", min: 2, resetStack: true },
+  /* 6 */ { id: "hlineto", min: 1, resetStack: true },
+  /* 7 */ { id: "vlineto", min: 1, resetStack: true },
+  /* 8 */ { id: "rrcurveto", min: 6, resetStack: true },
+  /* 9 */ null,
```

Comment:
These comments are definitely helpful, but can we perhaps add one space to make the complete list line-up better?
```suggestion
  /*  0 */ null,
  /*  1 */ { id: "hstem", min: 2, stackClearing: true, stem: true },
  /*  2 */ null,
  /*  3 */ { id: "vstem", min: 2, stackClearing: true, stem: true },
  /*  4 */ { id: "vmoveto", min: 1, stackClearing: true },
  /*  5 */ { id: "rlineto", min: 2, resetStack: true },
  /*  6 */ { id: "hlineto", min: 1, resetStack: true },
  /*  7 */ { id: "vlineto", min: 1, resetStack: true },
  /*  8 */ { id: "rrcurveto", min: 6, resetStack: true },
  /*  9 */ null,
```